docs/observability-logging.md

```diff
@@ -0,0 +1,101 @@
+# OpenChoreo Observability
+
+OpenChoreo provides observability for both developers and platform engineers. This is provided through logs as well as metrics in the openchoreo setup. 
+
+## Setting up Observability Logging
+
+ Observability logs are provided through a combination of fluentbit and open search in the data plane. The option of setting up observability is set to false by default. If you need to setup observability in your data plane you can execute the following command.
+
+```
+helm upgrade --install choreo-dataplane oci://ghcr.io/openchoreo/helm-charts/choreo-dataplane \
+            --kube-context kind-choreo --namespace "choreo-system" --create-namespace --timeout 30m \
+            --set certmanager.enabled=false --set certmanager.crds.enabled=false \
+            --set observability.logging.enabled=true
+```
+
+## Configuring Observability Logging feature 
+By default all logs in the following namespaces are collected and routed to the dashboard.
+
+- choreo-system - this will capture all operanation components of the choreo dataplane
+
+- dp* - this will capture all application logs of components deployed in choreo
+
+The configurations for this is in the template files under 
+
+ - `/openchoreo/github/sk/choreo/install/helm/choreo-dataplane/templates` 
+and the values for these templates can be found at 
+ - `/openchoreo/github/sk/choreo/install/helm/choreo-dataplane/values.yaml`
+
+### Some configurations that could be fine tuned are;
+
+  - Specifying what to include and exclude when collecting 
+By default all logs in namespaces "choreo-system" and "dp-*". Opensearch and Fluent-bit logs are excluded.
```

Comment:
I guess we need a new line here, otherwise it will render to a single line and it makes this bit confusing.
```suggestion
  - Specifying what to include and exclude when collecting

By default all logs in namespaces "choreo-system" and "dp-*". Opensearch and Fluent-bit logs are excluded.
```